src/app/App.vue

```diff
@@ -1,13 +1,38 @@
 <template>
-	<div class="container max-width-1200 mx-auto">
-		<h1 class="text-center font-bold text-xl">App</h1>
+	<div class="container mx-auto">
+		<h1 class="text-center text-xl my-5 font-hairline">Here you can find some comics info!</h1>
+		<SearchBar />
+		<div v-if="areFetching">
```

Comment:
usually it's something like `isFetching` or just `loading`

Reply:
and if you don't need additional div in DOM, you can use `<template v-if="areFetching"></template>` for it